File: src/main/kotlin/com/itmo/microservices/demo/order/api/dto/Booking.java
package com.itmo.microservices.demo.order.api.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Set;
import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Booking {
    private UUID id;
    private Set<UUID> failedItems;
}

File: src/main/kotlin/com/itmo/microservices/demo/order/api/dto/Booking.kt
package com.itmo.microservices.demo.order.api.dto

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data
import lombok.NoArgsConstructor
import java.util.*

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
class Booking {
    private val id: UUID? = null
    private val failedItems: Set<UUID>? = null
}